recipes/ignition-math/all/conanfile.py

```diff
@@ -0,0 +1,68 @@
+import os
+
+from conans import CMake, ConanFile, tools
+
+
+class IgnitionMathConan(ConanFile):
+    name = "ignition-math"
+    license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://ignitionrobotics.org/libs/math"
+    description = " Math classes and functions for robot applications"
+    topics = ("ignition", "math", "robotics", "gazebo")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False]}
+    default_options = {"shared": False}
+    generators = "cmake", "cmake_find_package_multi"
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        version_major = self.version.split('.')[0]
+        os.rename(f"ign-math-ignition-math{version_major}_{self.version}", self._source_subfolder)
```

Comment:
this format is not support in the minimum python version CCI supports, we have to use `"".format()`

You can also use the https://docs.conan.io/en/latest/reference/tools.html#tools-version to get the major version